CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ set(ddesc "An object-oriented component library supporting radiation")
 string(APPEND ddesc " transport applications.")
 project( Draco DESCRIPTION ${ddesc} VERSION 7.8 LANGUAGES CXX C)
 
+# Export compile commands database for Clang tools
+set(CMAKE_EXPORT_COMPILE_COMMANDS TRUE)
```

Comment:
How much clutter (if any) or slow down does this option incur?  Should it default to FALSE?

In either case, I'm not sure this change belongs in this PR. Can you elaborate on why it is added?

Reply:
This command causes cmake to generate a file called `compile_commands.json` in the top level of the build directory. 

Clang tools use this file to look up paths and compile commands for each translation unit; it's a convenient way for tools like clang-tidy get all the compiler flags, include paths, and so on. 

Jayenne has had it for about two years.